docker-compose.dev.yml

```diff
@@ -26,8 +26,7 @@ services:
       - DB_HOST=db
       - WEB_SERVER_HOST=0.0.0.0
       - CRON_MODE=
-      - STUDY_CONFIG=stage-program
-      - DYNAMIC_CONFIG=dev-emulator-study
```

Comment:
Removing `DYNAMIC_CONFIG` is good, but it is not clear that you need to change `STUDY_CONFIG`.
Changing the `STUDY_CONFIG` will work for one set of tests (with dynamic labels) but not the other.

You need to revert this, and then change your testing to indicate this, like so:

### with the default settings

results are

```
....
```

### with the change (using `git diff`)

```
-       - STUDY_CONFIG=stage-program
+      - STUDY_CONFIG=dev-emulator-config
```

results are

```
...
```

Reply:
Reverted the change to include STUDY_CONFIG=stage-program.
Updated the above test scenario for automated notebook test, in accordance with above specification.